### DISCUSSION TITLE
how to swizzle this siderbarmenu components?

### DISCUSSION THREAD

**Discussion by Chasen-Zhang** [facebook/docusaurus#7339] [Q&A]
![image](https://user-images.githubusercontent.com/15354455/166882500-4425cbdc-d5c7-4410-b458-b4c8c966c6ab.png)
how to swizzle this sidebarmenu menu components?

**Comment by Josh-Cena** [comment]
We've recently split the components, so the answer depends on what exactly you need. You can start with swizzling `DocSidebar`, figuring out which subcomponents you need to change, and deleting the rest of them after you're done with your necessary changes.

**Comment by Chasen-Zhang** [comment]
![image](https://user-images.githubusercontent.com/15354455/166884995-b5fc96dc-9ba8-45d0-91da-4f36e60430b7.png)
Is that right？

**Reply by Josh-Cena** [reply]
It depends on what you need to do. Do you intend to wrap it, or eject it? Have you gone through the docs and identified what you need to do?

Generally, I'll recommend you start with ejecting, unless you know for sure that wrapping will suffice.
